### ISSUE TITLE
Segmentation fault on Julia-1.7 beta3

### ISSUE THREAD

**Issue by shikil** [oscar-system/Polymake.jl#351]
Polymake version 0.5.6

```
julia> using Polymake
[ Info: Precompiling Polymake [d720cf60-89b5-51f5-aff5-213f193123e7]
C++ exception while wrapping module Polymake: invalid subtyping in definition of PropertyValue with supertype Any

signal (11): Segmentation fault
in expression starting at none:0
jl_gc_pool_alloc at /buildworker/worker/package_linux64/build/src/gc.c:1211
jl_gc_alloc_ at /buildworker/worker/package_linux64/build/src/julia_internal.h:336 [inlined]
jl_gc_alloc at /buildworker/worker/package_linux64/build/src/gc.c:3278
_new_array_ at /buildworker/worker/package_linux64/build/src/array.c:127 [inlined]
_new_array at /buildworker/worker/package_linux64/build/src/array.c:193 [inlined]
jl_alloc_array_1d at /buildworker/worker/package_linux64/build/src/array.c:464
decode_backtrace at /buildworker/worker/package_linux64/build/src/stackwalk.c:299
jl_get_excstack at /buildworker/worker/package_linux64/build/src/stackwalk.c:354
#current_exceptions#67 at ./error.jl:149
current_exceptions at ./error.jl:149 [inlined]
current_exceptions at ./error.jl:149 [inlined]
_start at ./client.jl:497
jfptr__start_40864.clone_1 at /home/shikil/julia-e76c9dad42/lib/julia/sys.so (unknown line)
_jl_invoke at /buildworker/worker/package_linux64/build/src/gf.c:2245 [inlined]
jl_apply_generic at /buildworker/worker/package_linux64/build/src/gf.c:2427
jl_apply at /buildworker/worker/package_linux64/build/src/julia.h:1787 [inlined]
true_main at /buildworker/worker/package_linux64/build/src/jlapi.c:559
jl_repl_entrypoint at /buildworker/worker/package_linux64/build/src/jlapi.c:701
main at /buildworker/worker/package_linux64/build/cli/loader_exe.c:42
__libc_start_main at /lib64/libc.so.6 (unknown line)
_start at /home/shikil/julia-e76c9dad42/bin/julia (unknown line)
Allocations: 4871886 (Pool: 4870101; Big: 1785); GC: 3
ERROR: Failed to precompile Polymake [d720cf60-89b5-51f5-aff5-213f193123e7] to /home/shikil/.julia/compiled/v1.7/Polymake/jl_jZPq8d.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::IO, internal_stdout::IO, ignore_loaded_modules::Bool)
   @ Base ./loading.jl:1466
 [3] compilecache(pkg::Base.PkgId, path::String)
   @ Base ./loading.jl:1410
 [4] _require(pkg::Base.PkgId)
   @ Base ./loading.jl:1120
 [5] require(uuidkey::Base.PkgId)
   @ Base ./loading.jl:1013
 [6] require(into::Module, mod::Symbol)
   @ Base ./loading.jl:997
```

**Comment by benlorenz** [comment]
Thanks for the report, we are aware of that error and a fix is in preparation at https://github.com/oscar-system/libpolymake-julia/pull/18

**Comment by shikil** [comment]
Fixed after update libpolymake_julia_jll to v0.4.117+0, thanks.
